Source/Core/DolphinQt2/GameList/GameFile.cpp

```diff
@@ -50,13 +50,12 @@ GameFile::GameFile(const QString& path) : m_path(path)
 
   if (!TryLoadCache())
```

Comment:
I think the nesting makes the intent of ordering more clear, but it doesn't really matter to me - i just reduced what was already there.